packages/wallet/src/redux/indirect-funding/player-a/actions.ts

```diff
@@ -0,0 +1,26 @@
+import { WalletProcedure } from '../../types';
+import { Commitment } from 'fmg-core/lib/commitment';
+
+export const STRATEGY_APPROVED = 'WALLET.INDIRECT_FUNDING.STRATEGY_APPROVED';
+export const strategyApproved = (
+  channelId: string,
+  procedure: WalletProcedure.IndirectFunding,
+) => ({
+  type: STRATEGY_APPROVED as typeof STRATEGY_APPROVED,
+  channelId,
+  procedure,
```

Comment:
Does this need to be a parameter? Is this action used for any other procedure?

Reply:
an action without a procedure can't be routed -- that's the point of the `procedure` property, isn't it?

Reply:
Never mind, I see what is happening here.